hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/api/Matcher.kt

```diff
@@ -19,7 +32,6 @@ import uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesear
   JsonSubTypes.Type(value = PncMatcher::class, name = "PNC"),
 )
 interface Matcher {
-  val type: String
```

Comment:
this was effectively declared twice - once here and once in the `@JsonTypeInfo`.  This meant that the openapi generator got confused and didn't create the `type` in the children with an `override` modifier